native/src/feature/onboarding/component/CodeInput.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { Text } from "react-native";
+import {
+  CodeField,
+  Cursor,
+  useBlurOnFulfill,
+  useClearByFocusCell,
+} from "react-native-confirmation-code-field";
+import styles from "./styles";
+
+const CELL_COUNT = 6;
+type Props = {
+  onChangeText: (text: string) => void | undefined;
+};
+const CodeInput: React.FC<Props> = ({ onChangeText }) => {
```

Comment:
Espacio entre estas líneas. y la constante : 

```suggestion
const CELL_COUNT = 6;

type Props = {
  onChangeText: (text: string) => void | undefined;
};

const CodeInput: React.FC<Props> = ({ onChangeText }) => {
```